generators/Exercises/Sublist.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Generators.Input;
+using Generators.Output;
+using Humanizer;
+using System.Linq;
+
+namespace Generators.Exercises
+{
+    public class Sublist : GeneratorExercise
+    {
+        protected override HashSet<string> AddAdditionalNamespaces() => new HashSet<string>() { typeof(System.Collections.Generic.IList<int>).Namespace };
+
+        private UnescapedValue inputValues(int[] list)
```

Comment:
By convention C# methods start with upper case letters so this method should be named `InputValues` instead of `inputValues`.

There are also some minor simplifications you can do to make this less verbose resulting in:
```
        private static UnescapedValue InputValues(int[] list)
        {
            var template = list != null ? string.Join(", ", list) : "";
            return new UnescapedValue($"new List<int> {{ {template} }}".Replace("  ", " "));
        }
```